lib/tracker_api/resources/project.rb

```diff
@@ -24,8 +24,6 @@ class Project
       attribute :initial_velocity, Integer
       attribute :iteration_length, Integer
       attribute :kind, String
-      attribute :label_ids, Array[Integer]
-      attribute :labels, Array[TrackerApi::Resources::Label]
```

Comment:
These need to stay so that the eager loading of labels will work if the `fields` param is used.